cmake/macros/macro_deal_ii_add_test.cmake

```diff
@@ -534,32 +588,23 @@ function(deal_ii_add_test _category _test_name _comparison_file)
         TIMEOUT ${TEST_TIME_LIMIT}
         )
 
+      if(_shared_target)
+        set_tests_properties(${_test_full} PROPERTIES
+          FIXTURES_REQUIRED ${_test_executable_full}
+          )
+      endif()
+
       if(_exclusive)
         #
         # Ensure that the test is not executed concurrently with any other
         # tests.
         #
         set_tests_properties(${_test_full} PROPERTIES RUN_SERIAL TRUE)
 
-      elseif(NOT ENABLE_PERFORMANCE_TESTS)
+      elseif()
         #
-        # Limit concurrency of mpi tests. We can only set concurrency for
-        # the entire test, which includes the compiling and linking stages
-        # that are purely sequential. There is no good way to model this
-        # without unnecessarily restricting concurrency. Consequently, we
-        # just choose to model an "average" concurrency as one half of the
-        # number of MPI jobs.
-        #
-        if(_n_cpu GREATER 2)
-          math(EXPR _slots "${_n_cpu} / 2")
-          set_tests_properties(${_test_full} PROPERTIES PROCESSORS ${_slots})
-        endif()
```

Comment:
Why remove this?

Reply:
Right, but don't you still want the *execution* of the test to be marked up with a specific number of processors, even though the *compilation* of the test requires only one processor?

Reply:
But the `else()` statement does! :smiley: